sip-core/src/test/java/de/ikor/sip/foundation/core/actuator/health/EndpointHealthIndicatorTest.java

```diff
@@ -12,35 +12,60 @@
 
 class EndpointHealthIndicatorTest {
 
-  EndpointHealthIndicator endpointHealthIndicator;
-  Endpoint endpoint;
-  Health health;
-  private static final String ENDPOINT_NAME = "name";
+  private EndpointHealthIndicator endpointHealthIndicator;
+  private Endpoint endpoint;
+  private Health health;
+  private static final String ENDPOINT_URI = "uri";
```

Comment:
Well done for refactoring!
Pay attention to Java convention as well: static is declared before object attributes,